be1-go/channel/lao/mod.go

```diff
@@ -418,17 +437,17 @@ func (c *Channel) processRollCallObject(action string, msg message.Message) erro
 
 	senderBuf, err := base64.URLEncoding.DecodeString(sender)
 	if err != nil {
-		return xerrors.Errorf("failed to decode sender key: %v", err)
+		return xerrors.Errorf(keyDecodeError, err)
 	}
 
 	// Check if the sender of the roll call message is the organizer
 	senderPoint := crypto.Suite.Point()
 	err = senderPoint.UnmarshalBinary(senderBuf)
 	if err != nil {
-		return answer.NewErrorf(-4, "failed to unmarshal public key of the sender: %v", err)
+		return answer.NewErrorf(-4, keyUnmarshalError, err)
 	}
 
-	if !c.hub.GetPubkey().Equal(senderPoint) {
+	if !c.organizerKey.Equal(senderPoint) {
```

Comment:
So now we have the organizer key in the channel because the `hub.GetPubkey()` might give the public key of a witness?

Reply:
Exactly, as hub.GetPubkey() gives the public key of the front-end linked to the current back-end, that could be a witness

Reply:
Yes, exactly